biojava-structure/src/test/java/org/biojava/nbio/structure/TestAltLocs.java

```diff
@@ -208,15 +209,20 @@ private void ensureAllAtomsSameAltCode(Group groupInputAltLocGroup, Group inputM
 	@Test
 	public void test1AAC() throws IOException, StructureException{
 
+		AtomCache cache = new AtomCache();
+		cache.setUseMmCif(false);
+		StructureIO.setAtomCache(cache);
```

Comment:
This is a good case for my new `GlobalsHelper` class. Unfortunately I haven't gotten around to merging it from 4.2 into master yet. (assignto: @sbliven)